pkg/pb/piecestore.proto

```diff
@@ -58,14 +58,14 @@ message PieceStore {
 
   RenterBandwidthAllocation bandwidth_allocation = 1;
   PieceData piece_data = 2;
-  SignedMessage authorization = 3;
+  PayerBandwidthAllocation payer_allocation = 3;
```

Comment:
How do we handle migration here. We shouldn't probably use the same tag number. Alternatively maybe make a completely new endpoint? @jtolds?

Reply:
maybe at the beginning, we can detect backward incompatible changes? I found something like this https://github.com/nilslice/protolock

Reply:
sgtm